### PR TITLE
ca: EC keys, digest, wording, umask

### PR DESCRIPTION
**Pull request by MEschenbacher**

### CODE REVIEW

File: ca/config-files/root-ca.cnf [hunk 1]
```diff
@@ -31,7 +31,7 @@ cert_opt                = no_pubkey
 copy_extensions         = copy
 crl_extensions          = crl_ext
 default_crl_days        = 180
-default_md              = sha256
+default_md              = sha384
```
Comment (ypid) [inline ca/config-files/root-ca.cnf:34]: Whats wrong with SHA256?

Comment (MEschenbacher) [reply ca/config-files/root-ca.cnf:34]: I don't think there's currently anything wrong with sha256, however, the new Lets Encrypt Root and Intermediates use sha384 as well as this wikipedia page https://en.wikipedia.org/wiki/Key_size#Effect_of_quantum_computing_attacks_on_key_strength (with the NIST recommending sha384).

Comment (ypid) [reply ca/config-files/root-ca.cnf:34]: Thanks very much. Thanks explains it for me. For reference, the new Lets Encrypt Root cert: https://crt.sh/?id=3335562555

**Review by alainwolf — APPROVED** [review]

**Comment by MEschenbacher** [comment]
Thanks!

**Pull request merged**